rikaicontent.js

```diff
@@ -750,6 +750,14 @@ var rcxContent = {
 		sel.removeAllRanges();
 		sel.addRange(range);
 		tdata.selText = sel.toString();
+
+		if(window.rikaichan.config.ttsEnabled == 'true') {
+			var text = sel.toString();
+			if(typeof text === 'string' && text.length > 0) {
+				//console.log("Sending playTTS " + text);

```

Comment:
No need for this comment, I guess.

Reply:
Well, I guess the idea is it will be easy to debug later. probably we should add a debug mode which would allow us to only send logs without adding commented lines which is a code smell at work.

Shouldn't be so strict here though.